packages/edit-post/src/components/keyboard-shortcut-help-modal/config.js

```diff
@@ -116,9 +116,12 @@ const blockShortcuts = {
 		{
 			keyCombination: '/',
 			description: __( 'Change the block type after adding a new paragraph.' ),
-			/* translators: The forward-slash character. e.g. '/'. */
 			ariaLabel: __( 'Forward-slash' ),
 		},
+		{
+			keyCombination: primaryAlt( 'v' ),
+			description: __( 'Toggle block between Editor and Html' ),
```

Comment:
I agree with @gziolo about the wording. What do you think about something like "Toggle between editing the block visually and editing HTML"?

Also, a nitpick, but it'd be great to move this shortcut up above the 'Change the block type after adding a new paragraph.' shortcut so that it's grouped with other block shortcuts that use the primary-alt modifier. 😄